comp/metadata/inventoryagent/inventoryagentimpl/inventoryagent.go

```diff
@@ -238,6 +252,37 @@ func (ia *inventoryagent) initData() {
 	ia.data["system_probe_protocol_classification_enabled"] = getBoolSysProbe("network_config.enable_protocol_classification")
 	ia.data["system_probe_gateway_lookup_enabled"] = getBoolSysProbe("network_config.enable_gateway_lookup")
 	ia.data["system_probe_root_namespace_enabled"] = getBoolSysProbe("network_config.enable_root_netns")
+
+	ia.data["feature_dynamic_instrumentation_enabled"] = getBoolSysProbe("dynamic_instrumentation.enabled")
+
+	ia.refreshMetadata()
```

Comment:
Why do we add this indirection rather than calling `fetchSecurityAgentMetadata` directly?

Reply:
We're going to do the same for all other processes. `refreshMetadata` is going to be expended very soon